src/legacy/core_plugins/kibana/server/tutorials/apm/envs/on_prem.js

```diff
@@ -50,6 +50,12 @@ export function onPremInstructions(apmIndexPattern) {
         title: i18n.translate('kbn.server.tutorials.apm.apmServer.title', {
           defaultMessage: 'APM Server',
         }),
+        callOut: {
+          title: 'Important: Updating to 7.0 or higher',
+          children:
+            'Please make sure your APM Server is updated to 7.0 or \
+             higher. You can also migrate your 6.x data with the Migration Assistant.',
+        },
```

Comment:
Major restriction: this can only return a string, since it's rendered server-side. Gotta figure out what we do with the link.

Reply:
Maybe for now we can say "with the migration assistant found in Kibana's management section", so they at least have some info on how to find it (if that's where it is)

Reply:
Done